### PR TITLE
fix user account api call

### PR DESCRIPTION
**Pull request by ajaypratap003**
Add url dependency to fix this issue.
https://issues.redhat.com/browse/MGDSTRM-8022

### CODE REVIEW

**Review by riccardo-forina — APPROVED** [review]
While I can be ok with adding the dependency, I suggest fixing the root issue. 

The codebase doesn't make use of this dependency directly. I assume it's one of the sdks that uses it? The rbac one? That sdk should list `url` as a peer dependency, so dependents would get a warning from npm for missing peer dependencies.

If we were to add a tool to automatically remove unused dependencies, we would have the same issue again (I used a tool like that to remove unused dependencies, reason why `url` is gone).

File: package.json [hunk 1]
```diff
@@ -128,7 +129,8 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-router-dom": "5.2.1",
-    "redux-promise-middleware": "6.1.2"
+    "redux-promise-middleware": "6.1.2",
+    "url": "0.11.0"
```
Comment (riccardo-forina) [inline package.json:133]: This is not what I meant. My point was to go on the repo of the SDK that requires `url` as a dependency and add it there as a peer dep. Not here.

**Comment by ajaypratap003** [comment]
> While I can be ok with adding the dependency, I suggest fixing the root issue.
> 
> The codebase doesn't make use of this dependency directly. I assume it's one of the sdks that uses it? The rbac one? That sdk should list `url` as a peer dependency, so dependents would get a warning from npm for missing peer dependencies.
> 
> If we were to add a tool to automatically remove unused dependencies, we would have the same issue again (I used a tool like that to remove unused dependencies, reason why `url` is gone).

I've raised this issue on slack channel with clouddot team and asked them to add url as peerDependency but we can't wait as it's critical issue.

File: package.json (hunk 1 above)
Comment (ajaypratap003) [reply package.json:133]: ok. 
Will tool remove unused dependency `url` ?

**Comment by riccardo-forina** [comment]
I know, we can merge the original commit which I approved. My comment was about doing something extra about this.

File: package.json (hunk 1 above)
Comment (riccardo-forina) [reply package.json:133]: There is no automated tool to check deps, yet. Just remove the peer you added and leave the dependencies one

Comment (ajaypratap003) [reply package.json]: ok. done

**Pull request merged**